.github/scripts/sync_labels.py

```diff
@@ -48,12 +48,13 @@ def get_linked_pr_from_issue_number(repo, number):
 
 def get_linked_issues_based_on_pr_body(repo, number):
     pr = repo.get_pull(number)
-    pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
-    matches = re.findall(pattern, pr.body)
+    pattern = rf'(?:fix(?:|es|ed)|resolve(?:|d|s)|close(?:|s|d))?(?:{repo}|https://github.com/{repo}/issues/)?(\d+)'
+    matches = re.findall(pattern, pr.body, re.IGNORECASE)
     if not matches:
         raise RuntimeError("No regex matches found in the body!")
     issue_number_from_pr_body = []
     for match in matches:
+        match = next((num for num in match if num), None)
```

Comment:
```py
import re
repo = 'scylladb/scylladb'
pattern = rf'(?:fix(?:|es|ed)|resolve(?:|d|s)|close(?:|s|d))?(?:{repo}|https://github.com/{repo}/issues/)?(\d+)'
body = 'Fixes: #10985\nFixes: scylladb/scylla-enterprise#2539\nFixes https://github.com/scylladb/scylladb/issues/17025'
print(re.findall(pattern, body, re.IGNORECASE))
```
```
['10985', '2539', '17025']
```

i am testing with the above test case. could you share the test you've been using?